### PR TITLE
Add Section 1.2 Languages

### PR DESCRIPTION
**Pull request by behnam**
Add details of the languages covered by the document.

Closes <https://github.com/w3c/alreq/issues/96>

### CODE REVIEW

**Comment by behnam** [comment]
Preview: https://rawgit.com/behnam/alreq/intro/index.html#languages

File: index.html [hunk 1]
```diff
@@ -159,6 +159,51 @@
...
+        </h4>
+
+
+        <p><dfn>Standard Arabic</dfn>—a.k.a. Modern Standard Arabic or Literary Arabic—is the
+        standardized and literary variety of Arabic used in writing and in most formal speech in
+        countries of Northern Africa and West Asia. Regional and classical dialects of Arabic may
+        differ in layout and text details and are <em>not</em> covered by this document.</p>
+
+
+        <p>However, there are some major differences in common pratices between the <dfn>Western
```
Comment (r12a) [inline index.html]: practices

File: index.html [hunk 2]
```diff
@@ -159,6 +159,51 @@
...
+        Arab regions</dfn>—that is North-West Africa—and <dfn>Eastern Arab regions</dfn>—which is
+        North-East Africa and West Asia. For example, the numeral digits used in the two regions
+        and their formatting are vastly different. Although, there's no clear line between the
+        Eastern and Western Arab regions.</p>
+      </section>
+
+
+      <section id="persian_langauge">
+        <h3><a href="#persian_langauge">Persian Language</a>
```
Comment (r12a) [inline index.html]: langauge -> language

File: index.html [hunk 3]
```diff
@@ -159,6 +159,51 @@
...
+        </h3>
+
+
+        <p><dfn>Persian</dfn>—a.k.a. Modern Persian—is the standardized and literary variety of the
+        languages as used as official language in Iran and Afghanistan. The dialect of Persian is
```
Comment (r12a) [inline index.html]: "the standardized and literary variety of the languages as used as official language in Iran and Afghanistan"
-> (?)
the standardized and literary variety of the official languages used in Iran and Afghanistan

File: index.html [hunk 4]
```diff
@@ -159,6 +159,51 @@
...
+        Iran is also called Western Persian, and is locally known as <span class=
+        "qterm">Farsi</span>. The dialect of Persian in Afghanistan is also known as Eastern
+        Persian, and is locally known as Dari.</p>
+      </section>
+
+
+      <p><dfn>Tajiki</dfn>, as the language is called in Tajikistan, is written in the Cyrillic
```
Comment (r12a) [inline index.html]: Maybe add '—a.k.a Tajik, or Tajiki Persian—' ?

**Pull request merged**